tests/acceptance/welcome/payment-info-test.js

```diff
@@ -13,14 +13,25 @@ function visitPaymentInfoWithApp(options, userData){
 
   signInAndVisit(url, userData);
   andThen(function(){
-    if (!options) { return clickButton('Skip this step'); }
+    if (!options) {
+      return clickButton('Skip this step');
+    }
 
-    if (options.dbType){ click(`.${options.dbType} a`); }
+    if (options.dbType) {
+      click(`.${options.dbType} a`);
+    }
+    if (options.dbInitialDiskSize) {
+      andThen(function(){
+        Ember.$('.slider').trigger('slide', options.dbInitialDiskSize);
+      });
+    }
     for (var prop in options){
       let dasherized = prop.dasherize();
 
-      // db-type is a button, not a fillIn-able input
-      if (dasherized === 'db-type') { continue; }
+      // non-fillIn-able inputs
+      if ('db-type db-initial-disk-size'.w().indexOf(dasherized) !== -1) {
```

Comment:
funny that this likely works the same with and without the `w()`. It's more correct this way, though.